i18n/fr/docusaurus-plugin-content-docs/version-22.04/migrate/migrate-from-el-to-debian.md

```diff
@@ -0,0 +1,227 @@
+---
+id: migrate-from-el-to-debian
+title: Migrer depuis un OS de type EL vers Debian 11 (vers un Centreon 22.04)
+---
+
+## Prérequis
+
+Cette procédure ne s'applique que dans les conditions suivantes :
+
+- Vous souhaitez migrer d'un OS de type EL 64-bits vers Debian 11. Par exemple, vous souhaitez migrer d'un CentOS 7 à Debian 11.
+- Votre version de Centreon est 18.10 ou plus récente, et vous souhaitez paser à Centreon 22.04 (Debian est uniquement supporté depuis la version 22.04). Si vous souhaitez migrer depuis une plus ancienne version, [contactez l'équipe support Centreon](https://centreon.force.com).
+
+> En cas de migration d'une plateforme disposant du système de redondance
+> Centreon, il est nécessaire de contacter le
+> [support Centreon](https://centreon.force.com).
+
+## Migrer un serveur central
+
+### Étape 1 : Installer le nouveau serveur
+
+1. Installez votre nouvel OS en suivant les [prérequis](../installation/prerequisites.md).
+
+2. Installez un nouveau serveur central Centreon à partir des [paquets](../installation/installation-of-a-central-server/using-packages.md)
+   jusqu'à terminer le processus d'installation en vous connectant à l'interface web.
+
+   > Pendant le processus d'installation web, utilisez le même mot de passe pour l'utilisateur **centreon** que sur l'ancienne plateforme.
+
+3. Réalisez les mises à jour logicielle et système :
+
+```shell
+apt update && apt upgrade
+```
+
+### Étape 2 : Synchroniser les données
+
+1. Connectez-vous à votre ancien serveur Centreon.
+
+2. Générez une paire de clés ssh pour **root** :
+
+   ```shell
+   ssh-keygen -t rsa
```

Comment:
Key pair generated here will go, by default, to `/root/.ssh/id_rsa.pub and /root/.ssh/id_rsa`, maybe we should add the two possible case

- user chooses to use default values => follow procedure as described with id_rsa and id_rsa.pub
- user chooses a custom key pair name => add the rsync command with the proper -e + -i options
`rsync -avz -e "ssh -i /root/.ssh/id_rsa_custom" /etc/centreon root@10.25.26.27:/tmp/`